### ISSUE TITLE
Null type warning

### ISSUE THREAD

**Issue by willemneal** [MaxGraey/bignum.wasm#34]
```
WARNING AS226: Expression resolves to unusual type 'usize'.

     __multi3(null, a.lo, a.hi, b.lo, b.hi);
              ~~~~
 in ~lib/as-bignum/integer/u128.ts(434,13)
```

Does this just need a type cast?

**Comment by MaxGraey** [comment]
yeah. It wasn't a warning before. Going to fix this

**Comment by MaxGraey** [comment]
Fixed in 0.1.6
